### PR TITLE
feat: update info.xml and appversion

### PR DESCRIPTION
**Pull request by miaulalala**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/calendar/main/screenshots/week_sidebar.png</screenshot>
 	<dependencies>
 		<php min-version="7.4" max-version="8.3" />
-		<nextcloud min-version="25" max-version="28" />
+		<nextcloud min-version="26" max-version="29" />
```
[the remark this answers is not in the document]
Comment (miaulalala) [reply appinfo/info.xml:38]: are you being sarcastic or FR?

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply appinfo/info.xml:38]: I mean it. you are adding 29 and dropping 25 support with this commit/PR. the commit title does not reflect that

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply appinfo/info.xml:38]: https://stackoverflow.com/questions/26944762/when-to-use-chore-as-type-of-commit-message

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
CI tests 25 but not 28

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**